eigen/CMakeLists.txt

```diff
@@ -14,9 +14,9 @@ if(CONFIG_COMPILER_OPTIMIZATION_DEFAULT)
     set(opt_args -DCMAKE_BUILD_TYPE=Debug)
 elseif(CONFIG_COMPILER_OPTIMIZATION_SIZE)
     set(opt_args -DCMAKE_BUILD_TYPE=Relesee)
-elseif(COMPILER_OPTIMIZATION_PERF)
+elseif(CONFIG_COMPILER_OPTIMIZATION_PERF)
     set(opt_args -DCMAKE_BUILD_TYPE=Relesee)
```

Comment:
@kutukvpavel Could you please also fix another typo here (should be `Release`). And in the line above I believe it should be `MinSizeRel`

Reply:
@igrr No problem. Fixed.